.gitignore

```diff
@@ -8,6 +8,7 @@
 *.egg
 *.egg-info
 dist
+*/setup_base
```

Comment:
So I added `<Package ROOT>/setup_base/base.py`, but after building a subpackage it copies that to the subpackage root `<Package ROOT>/storage/setup_base/base.py`.

So `google-cloud-python/setup_base` is ok, but `google-cloud-python/storage/setup_base` should be ignored.